jmx-scraper/src/integrationTest/java/io/opentelemetry/contrib/jmxscraper/target_systems/TomcatIntegrationTest.java

```diff
@@ -46,67 +46,76 @@ protected JmxScraperContainer customizeScraperContainer(
   }
 
   @Override
-  protected void verifyMetrics() {
-    waitAndAssertMetrics(
-        metric ->
-            assertGaugeWithAttributes(
-                metric,
-                "tomcat.sessions",
-                "The number of active sessions",
-                "sessions",
-                attrs -> attrs.containsKey("context")),
-        metric ->
-            assertSumWithAttributes(
-                metric,
-                "tomcat.errors",
-                "The number of errors encountered",
-                "errors",
-                attrs -> attrs.containsOnly(entry("proto_handler", "\"http-nio-8080\""))),
-        metric ->
-            assertSumWithAttributes(
-                metric,
-                "tomcat.processing_time",
-                "The total processing time",
-                "ms",
-                attrs -> attrs.containsOnly(entry("proto_handler", "\"http-nio-8080\""))),
-        metric ->
-            assertSumWithAttributes(
-                metric,
-                "tomcat.traffic",
-                "The number of bytes transmitted and received",
-                "by",
-                attrs ->
-                    attrs.containsOnly(
-                        entry("proto_handler", "\"http-nio-8080\""), entry("direction", "sent")),
-                attrs ->
-                    attrs.containsOnly(
-                        entry("proto_handler", "\"http-nio-8080\""),
-                        entry("direction", "received"))),
-        metric ->
-            assertGaugeWithAttributes(
-                metric,
-                "tomcat.threads",
-                "The number of threads",
-                "threads",
-                attrs ->
-                    attrs.containsOnly(
-                        entry("proto_handler", "\"http-nio-8080\""), entry("state", "idle")),
-                attrs ->
-                    attrs.containsOnly(
-                        entry("proto_handler", "\"http-nio-8080\""), entry("state", "busy"))),
-        metric ->
-            assertGaugeWithAttributes(
-                metric,
-                "tomcat.max_time",
-                "Maximum time to process a request",
-                "ms",
-                attrs -> attrs.containsOnly(entry("proto_handler", "\"http-nio-8080\""))),
-        metric ->
-            assertSumWithAttributes(
-                metric,
-                "tomcat.request_count",
-                "The total requests",
-                "requests",
-                attrs -> attrs.containsOnly(entry("proto_handler", "\"http-nio-8080\""))));
+  protected MetricsVerifier createMetricsVerifier() {
+    return MetricsVerifier.create()
+        .add(
+            "tomcat.sessions",
+            metric ->
+                metric
+                    .hasDescription("The number of active sessions")
+                    .hasUnit("sessions") // TODO: not aligned with semconv. Should be "{session}"
```

Comment:
Like in the previous PRs where such units inconsistencies were found I think it's worth fixing them straight away as they aren't "breaking changes" most of the time and we will have to fix it at some point because it's now part of the stable semantic conventtions.

Reply:
So my understanding is that in this case it may be a kind of breaking change. This is not the annotation in curly braces that is supposed to be skipped by parser.
I've been hesitating if I should update it or not and finally decided to leave it, but if reviewers agree that it should be changed then I'll gladly update it.

Reply:
I think it's ok to update it since we are updating units elsewhere (in OTel we've said changing units is breaking whether they're curly braces or not)

Reply:
Great! I'll make the fix then